src/mixins/planningActionMixin.js

```diff
@@ -554,12 +560,13 @@ export const planningActionMixin = {
           }
 
           this.editionModal = false;
+          this.loading = false;
           NotifyPositive('Évènement supprimé.');
         }).onCancel(() => NotifyPositive('Suppression annulée'));
       } catch (e) {
```

Comment:
`try/catch/finally` à mettre ds le `onOk`. Et ddu coup, plus besoin du `this.loading = false` ds le `onOk`